libs/frontend/feature-build/src/lib/style/use-reusable-style-menu.ts

```diff
@@ -135,18 +135,20 @@ export const useReusableStyleMenu = (): IUseStyleMenuFeature => {
       activeBreakpoint.value,
     )
 
-    return Object.entries(flattenedStyles).map(
-      ([css, source]) =>
-        ({
-          pseudoClass: currentPseudoClass.value,
-          property: css as Css,
-          value: source.value,
-          sourceType: source.sourceType,
-          sourceId: source.sourceId,
-          sourceBreakpointId: source.sourceBreakpointId,
-          inheritedFrom: getInheritedFrom(source),
-        }) as IInheritedStyleEntry,
-    )
+    return Object.entries(flattenedStyles)
+      .sort((a, b) => a[0].localeCompare(b[0]))
```

Comment:
@abemscac I think including the sort function here (similar to component/override styles) solves the problem, but I might have overlooked something.